cloudfoundry-client/src/main/lombok/org/cloudfoundry/client/v2/serviceplans/ListServicePlansRequest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.cloudfoundry.client.v2.serviceplans;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Singular;
+import lombok.ToString;
+import org.cloudfoundry.client.QueryParameter;
+import org.cloudfoundry.client.Validatable;
+import org.cloudfoundry.client.ValidationResult;
+import org.cloudfoundry.client.v2.InFilterParameter;
+import org.cloudfoundry.client.v2.PaginatedRequest;
+
+import java.util.List;
+
+/**
+ * The request payload for the List Service Plans operation
+ */
+@Data
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public final class ListServicePlansRequest extends PaginatedRequest implements Validatable {
+
+    /**
+     * The active flag
+     *
+     * @param active the active flag
+     * @return the active flag
+     */
+    @Getter(onMethod = @__(@QueryParameter("active")))
```

Comment:
@antechrestos Until documented (or a bug is opened against the documentation) we assume that all of the filters are `@InFilterParameter`.  If it's not, it'll get caught when we integration test the implementations.